tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -59,6 +60,15 @@ static DatadogLogging()
                     // Do nothing when an exception is thrown for attempting to access the filesystem
                 }
 
+                try
```

Comment:
Open questions for discussion: 
Is it possible to call that in a a fire and forget task to avoid the blocking call (as initialising logging should happen fast) ? Also, have you thought of what happens if several process are launched in //? 
Should we handle the fact that a process could stay alive for several weeks?

Reply:
Moved the method call to a fire and forget task.